requirements.txt

```diff
@@ -1,4 +1,5 @@
 anyconfig~=0.9.7
+avro==1.10.0
```

Comment:
Can you move this to `test_requirements.txt` instead? Avro isn't needed to use the core functionality of Kedro. And would it be possible to pin it more loosely instead of strictly to 1.10.0?

Reply:
What is the purpose of the `test_requirements.txt` file?

Reply:
@kislerdm `test_requirements.txt` is used to specify an extended set of dependencies required to run the full unit test suite, among other things. If you look at [the first line](https://github.com/quantumblacklabs/kedro/blob/master/test_requirements.txt#L1), you'll see that it includes all of `requirements.txt`.

While Avro isn't core to the framework, you'll also need to add lines in [setup.py](https://github.com/quantumblacklabs/kedro/blob/master/setup.py) to include the appropriate Avro dependencies when somebody wants to use the dataset.